kazakh/ualeikum.md

```diff
@@ -0,0 +1 @@
+Баяғы бір заманда бай мен кедей көрші өмір сүріпті. Олардың араздығы сондай, бірін-бірі көргісі келмейді екен. [Байдың үйі](bai/bai.md) кең, әдемі болыпты. Іші толған – алтын мен күміс. Ал кедейдің үйі ағаштан жасалған, төбесін бұтамен жапқан қора екен. Алтын мен күміс түгілі, нанын әзер тауып жепті. Егін егіп, оны суарып, күтіп, орып, өз күнін өзі көріпті.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.